### ISSUE TITLE
Character counter pushes breaks layout

### ISSUE THREAD

**Issue by ivanBell199** [materializecss/materialize#191]
## Expected Behavior
`<span>` that is added by character counter should not break the layout.

## Current Behavior
It pushes the input below to the bottom.

This:
![image](https://user-images.githubusercontent.com/65890952/133930987-cac5c0f4-d9da-4649-b1b0-89226642aa53.png)
becomes this:
![image](https://user-images.githubusercontent.com/65890952/133931007-f082e8df-3910-4463-a76f-966e173b4e5b.png)

## Steps to Reproduce (for bugs)
1. Create a number of text inputs with .col.s6 so that there are two inputs per line.
2. Add character counter to inputs.
3. Click inside the first input.

Here is a [codepen](https://codepen.io/iBeliakov/pen/bGRvvbP)

## Your Environment
Materialize 1.1.0-alpha

**Comment by ivanBell199** [comment]
I noticed that this problem is solved if `.row` uses `flex`. So the issue will likely be solved automatically in the next version of the framework when we use `flex` for grid system.

**Comment by LoganTann** [comment]
Yes, using flexbox fixes this, but I found 2 others workaround : 

the CSS quickfix (dirty but works) : 
```css
 .character-counter {
    float: initial;
    font-size: 12px;
    position: absolute;
    bottom: -8px;
    right: 8px;
}
```

the HTML quickfix (in case you can change adopt this structure and you don't want to use CSS) : 
```html
<div class="row">
  <form class="col s12">
    <div class="row">
      <div class="input-field col s6">
        <input id="input_text1" type="text" data-length="10">
        <label for="input_text1">Input text</label>
      </div>
      <div class="input-field col s6">
        <input id="input_text2" type="text" data-length="10">
        <label for="input_text2">Input text</label>
      </div>
    </div>
    <div class="row">
      <div class="input-field col s6">
        <input id="input_text3" type="text" data-length="10">
        <label for="input_text3">Input text</label>
      </div>
      <div class="input-field col s6">
        <input id="input_text4" type="text" data-length="10">
        <label for="input_text4">Input text</label>
      </div>
    </div>
  </form>
</div>
````

Also, I couldn't reproduce this issue in Firefox. Strangely, it appears only in chrome.

idk if we should close this issue ?
